doc/dev/developer_guide/dash-devel.rst

```diff
@@ -1026,6 +1026,59 @@ Alerts and notifications
 Default notification should have `text-info` color. Success notification should
 have `text-success` color. Failure notification should have `text-danger` color.
 
+Accessibility
```

Comment:
I would add this right after chapter `i18n` (and would welcome a name change to `Internationalization`), as both concerns pursue a similar goal: to reach a broader user base by lowering barriers to entry (whether cultural or cognitive).

Reply:
In any case, I also understand the rationale for putting this under the "UI Style guide". We could add a link there to an outer accessibility section. I was just trying to give "accessibility" the visibility it deserves, and nesting it inside the "UI Style guide" diminished its imporance (IMHO).

Reply:
Thanks @epuertat